src/frontends/onnx/frontend/src/input_model.cpp

```diff
@@ -453,7 +453,8 @@ void InputModel::add_tensor_names(std::shared_ptr<Model>& model) {
         return std::find_if(std::begin(model_inputs),
                             std::end(model_inputs),
                             [&name](const OutputVector::value_type& input) {
-                                return input.get_names().count(name) > 0;
+                                return std::find(input.get_names().begin(), input.get_names().end(), name) !=
+                                       input.get_names().end();
```

Comment:
should not we add some helpers like `ov::util::contains` (or copy https://github.com/openvinotoolkit/openvino/blob/master/src/inference/dev_api/ie_algorithm.hpp#L32-L35) to https://github.com/openvinotoolkit/openvino/blob/master/src/common/util/include/openvino/util/common_util.hpp ?

Reply:
Maybe Tensor::has_name?

Reply:
ok, I agree